src/App/App.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+
+import { Redirect, Route, Router, Switch, useLocation } from "wouter";
+
+import "@styles/style.scss";
+import OrgRepositoriesPage from "./pages/org_repositories_page";
+import RepositoryPage from "./pages/repository_page/RepositoryPage";
+
+function App() {
+  const [pathname] = useLocation();
+
+  React.useEffect(() => {
+    window.scrollTo(0, 0);
+  }, [pathname]);
+
+  return (
+    <main>
+      <Router>
+        <Switch>
+          <Route path="/org/:org" component={OrgRepositoriesPage} />
+          <Route path="/repository/:org/:repoName" component={RepositoryPage} />
```

Comment:
Немного не понял такой выбор роутов
мне кажется проще было бы сделать 
```suggestion
          <Route path="/repositories/:org" component={OrgRepositoriesPage} />
          <Route path="/repositories/:org/:repoName" component={RepositoryPage} />
```
так по ссылке понятно что ты попал из списка репозиториев на конкретный репозиторий

Reply:
Да, согласен, подправлю.